src/app/components/side-nav/side-nav.component.spec.ts

```diff
@@ -26,7 +34,14 @@ describe('SideNavComponent', () => {
       ],
       declarations: [SideNavComponent],
       providers: [
-        { provide: EXAMPLE, useClass: MockExample, multi: true },
+        ...exampleSpies.map((example) => ({ provide: EXAMPLE, useValue: example, multi: true })),
+        { provide: Router, useValue: routerSpy },
+        {
+          provide: RuntimeService, useValue: MockService(RuntimeService, {
```

Comment:
to remove having to specify generic for `of` you could create a partial variable, i.e.

```typescript
const runtimeSvcOverriddes: Partial<RunTimeService> = {
  orientation$: of('landscape'),
  mediaSize$: of('large'),
}
```

And then:

```suggestion
          provide: RuntimeService,
          useValue: MockService(RuntimeService, runtimeSvOverrides),
```

Reply:
Updated